### PR TITLE
prep repo for testing in jestj

### PR DESCRIPTION
**Pull request by cpprian**
+ Change coverage command
+ Update jest config for react-scripts

collectCoverageFrom -> We want to test only feature components, not configurations or types
coverageThreshold -> It enables you to define minimum acceptable coverage levels for different types of code
preset -> Helps you define js/ts syntax, for react-scripts doesn't work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,5 +65,19 @@
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "prettier": "^2.8.4"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.{js,jsx,ts,tsx}",
+      "!src/**/*.{stories,story,spec,test}.{js,jsx,ts,tsx}"
```
Comment (kasia251011) [inline package.json:72]: I would reccomend you to only test component and utlilities folders:
```
      "src/components/**/*.{js,jsx,ts,tsx}",
      "src/utilities/**/*.{js,jsx,ts,tsx}"
```

There is no need (or sometimes there is no possibility) to check all files, for eample constants.

File: package.json [hunk 2]
```diff
@@ -64,5 +65,19 @@
...
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 80,
+        "functions": 80,
+        "lines": 80,
+        "statements": -10
+      }
+    }
```
Comment (kasia251011) [inline package.json:81]: Why are they set like that? What you wanted to achieve? What is the default config?

Comment (kasia251011) [reply package.json:81]: For me without this configuration, it look fine

**Review by kasia251011 — CHANGES_REQUESTED** [review]
You dont have to explain what those fields means - but you rather should explain why you set them to those values - what you wanted to achieve. For now just answer for my comments.

File: package.json (hunk 2 above)
Comment (cpprian) [reply package.json:81]: with coverageThreshold
Screenshot 2023-04-12 at 13 42 40
without coverageThreshold 
Screenshot 2023-04-12 at 13 43 47

It gives you a better view of test coverage

File: package.json (hunk 1 above)
Comment (cpprian) [reply package.json:72]: So, we can make small changes: 
+ "src/{components, utilities}/*.{js,jsx,ts,tsx}",

during testing, we can also make some changes if necessary

File: package.json (hunk 2 above)
Comment (kasia251011) [reply package.json:81]: Are you sure you want to be so restricted for now? If you really want to be so perfectionist, make it 100 % :) If not, I would reccomend delete those. But after all, its just an opinion, so you can keep this if you want.

Comment (kasia251011) [reply package.json:81]: Are you sure you want to be so restricted for now? If you really want to be so perfectionist, make it 100 % :) If not, I would reccomend delete those. But after all, its just an opinion, so you can keep this if you want.

**Pull request closed without merging**